content/tutorials/getting-started/nlb-environment.md

```diff
@@ -0,0 +1,136 @@
+sensenet__Clients__adminui__RepositoryHosts__1__PublicHost="https://host2.example.com"
+```
+
+For more information about configuring the Identity Server, see the [Identity Server configuration](/api-docs/configuration/identityserver) article.
+
+### Search Service
+The search service requires an access to the repository database. It also needs to be able to receive indexing and search requests from the repository instances. The following sections describe how to configure the different components of the search service.
+
+Please note that the MessageExchange value must be the same as the one configured for the repository instances. It also has to be unique for each sensenet application in case you are using the same RabbitMQ service for multiple sensenet applications.
+
+```json
+{
+  "ConnectionStrings": {
+    "SecurityStorage": "..."
+  },
+  "sensenet": {
+    "security": {
+      "rabbitmq": {
+        "ServiceUrl": "amqps://user:password@host:5671/",
+        "MessageExchange": "snsecurity-myapp"
+      }
+    }
+  }
+}
+```
+
+The same configuration in an environment variable format:
+
+```txt
+ConnectionStrings__SecurityStorage="..."
+sensenet__security__rabbitmq__ServiceUrl="amqps://user:password@host:5671/"
+sensenet__security__rabbitmq__MessageExchange="snsecurity-myapp"
+```
+
+#### Authentication
+The Search Service is an internal service that is not accessible from the outside world. It is only used by the repository to provide indexing and search functionality. It does not require any authentication and must not be accessible from outside.
+
+### sensenet repository instances
+In a Network Load Balanced environment you'll have multiple repository instances. These instances need to be configured to work together. This section describes how to configure the different features of the repository service.
+
+Please note that you have to configure RabbitMQ for sending system and security messages in two different places. The security *MessageExchange* value must be the same as the one configured for the search service. The general message exchange value (dedicated to system messages) has to be *identical on all sensenet instances*, but must be unique for the application.
+
+You also have to provide the address for the Search service so that the repository can send indexing and query requests through gRPC.
+
+```json
+{
+  "ConnectionStrings": {
+    "SnCrMsSql": "..."
+  },
+  "sensenet": {
+    "security": {
+      "rabbitmq": {
+        "ServiceUrl": "amqps://user:password@host:5671/",
+        "MessageExchange": "snsecurity-myapp"
+      }
+    },
+    "rabbitmq": {
+      "ServiceUrl": "amqps://user:password@host:5671/",
+      "MessageExchange": "snmessaging-myapp"
+    },
+    "search": {
+      "service": {
+        "ServiceAddress": "https://searchservice.example.com",
+        "ValidateServerCertificate": false,
+        "ServiceWritePartitionSize": 30
+      }
+    }
+  }
+}
+```
+
```

Comment:
```"ValidateServerCertificate": false,```: because all config will be copied, it should be true and it should be explained that "false" is only a development tool.